tests/tests.js

```diff
@@ -1,10 +1,108 @@
-const { expect } = require('chai')
-const { describe, it } = require('mocha')
+const chai = require('chai')
+const {
+  describe, it, beforeEach, afterEach,
+} = require('mocha')
 const employee = require('./employee')
 const products = require('./products')
 const pricing = require('../pricing')
+const sinon = require('sinon')
+const sinonChai = require('sinon-chai')
+
+const { expect } = chai
+chai.use(sinonChai)
+
+
+describe('pricing', () => {
+  describe('formatPrice', () => {
+    it('returns a price with two decimal places not rounded when given a number with multiple decimal places', () => {
+      const formattedPrice = pricing.formatPrice(15.1566663)
+
+      expect(formattedPrice).to.equal(15.15)
+    })
+  })
+
+
+  describe('getEmployerContribution', () => {
```

Comment:
`getEmployerContribution` works differently for dollar based contributions and percentage based. Looking at the function I would expect two tests here so you exercise both branches.